kalendar-foundation/detekt.yml

```
@@ -1,35 +1,35 @@
-  Indentation:
-    active: false
-
-style:
-  MaxLineLength:
-    active: true
-    maxLineLength: 166
-  MagicNumber:
-    active: false
-  DestructuringDeclarationWithTooManyEntries:
-    active: true
+complexity:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)